packages/react/src/SubdomainNavBar/SubdomainNavBar.module.css

```diff
@@ -220,7 +220,7 @@
 }
 
 @media screen and (max-width: 767px) {
-  .SubdomainNavBar-link:first-child {
+  .SubdomainNavBar-link:first-of-type {
```

Comment:
Nice catch! Any regressions from this? Guessing not 🤔

Reply:
There seems to be a test breaking due to this change.

Reply:
But it isn't breaking locally. It said accessibility issue in description within the breaking CI pipeline